Documents/speedAlgorithm.md

```diff
@@ -28,43 +28,4 @@ So you may be wondering how this equation works. Well I can tell you. For the sp
     
   The equation is made on a pretty simple idea. The Strength, friction, and current speed are all negative effects that increase your speed. If you want to have a better or worse speed, then you should change the strength, or the current speed of the animal. The friction cannot be changed as it is part of the current environment you are fighting in.  
     
-  If you want to make the animal get hit less, then you would increase the weight of the animal. With this you have to be extra careful though, because if the difference between the negatives and the positives isn't greater than two, the animal will dodge every attack.
-
-### Hawk:
-- S = 30
-- W = 70  
-Desert Speed: 2.  
-Beach speed:	55.
-
-### Bull: 
-- S = 29
-- W = 500  
-Desert Speed: 45.  
-Beach speed: 110.  
-
-### Bear:
-- S = 10
-- W = 113  
-Desert Speed: 22.  
-Beach speed: 35.  
-
-### Lion:
-- S = 16
-- W = 111  
-Desert Speed: (place speed).  
-Beach speed: 21.  
-
-### Crocodile:
-- S = 22
-- W = 102  
-Desert Speed: 100.  
-Beach Speed:
-### Elephant:
-- S = 18
-- W = 515  
-Desert Speed: 60.
-### Hyena: 
-- S = 7
-- W = 40  
-Desert Speed: 5.  
-
+  If you want to make the animal get hit less, then you would increase the weight of the animal. With this you have to be extra careful though, because if the difference between the negatives and the positives isn't greater than two, the animal will dodge every attack.
```

Comment:
```suggestion
  If you want to make the animal get hit less, then you would increase the weight of the animal. With this you have to be extra careful though, because the animal will dodge every attack **unless the difference between the negatives and the positives is greater than two**.
```